tests/robustness/report/wal.go

```diff
@@ -109,23 +125,23 @@ func persistedRequests(lg *zap.Logger, dataDirs []string, reader persistedReques
 		}
 	}
 	// Select longest history that has votes from quorum.
-	longestHistory := []model.EtcdRequest{}
-	quorum := len(dataDirs)/2 + 1
+	longestHistory := []raftpb.Entry{}
+	quorum := len(memberEntries)/2 + 1
 	foundQuorum := false
-	for i := 0; i < len(memberRequestHistories); i++ {
+	for i := 0; i < len(memberEntries); i++ {
```

Comment:
nit
```suggestion
	for i := range len(memberEntries) {
```

Reply:
This would be unrelated change that makes PR harder to read.